crates/macros/src/lib.rs

```diff
@@ -19,6 +19,9 @@ use syn::{
 struct CrossTestData {
     /// imlementations
     impls: ExprArray,
+    /// builder impl
+    #[builder(default = "syn::parse_str(\"[SimpleBuilderImplementation]\").unwrap()")]
```

Comment:
I slightly prefer not having a default implementation (and just explicitly writing `SimpleBuilderImplementation` in all the integration tests), but I think it's fine either way and we could just change later if we want